### ISSUE TITLE
Ongoing discussions

### ISSUE THREAD

**Issue by grabbou** [reasonml-community/bs-react-native#245]
Hey,

We are working on some improvements to `bs-react-native` and wanted to work with everyone that has ever contributed to this repository on these. It would be great to discuss some issues, future roadmap and where we are headed.

I was thinking about creating a Discord group/channel where we could discuss `bs-react-native` related work.

Do you have Discord? If yes, please send me your id here and I will invite you all to the channel.

**Comment by cknitt** [comment]
Hi, please add me to the channel. My Discord id is `cknittel#6651`. Thanks!

**Comment by grabbou** [comment]
No reason in particular. General conclusion we have reached with Jordan after debating where to put a new channel. There's still `#react` channel on Discord that is designated for general troubleshooting and inquiries, as per our documentation. This one was targeted at the contributors, specifically regarding ongoing development.

Since we have reached the limit of a group on Discord (FYI, it's 10 people), we are going to create a channel and it will be referenced in the `contributing` section. 

I will update this issue once we do that.
